oct/ansible/oct/callback_plugins/log_results.py

```diff
@@ -380,24 +350,7 @@ def v2_runner_on_async_failed(self, result):
 
         :param result: result of the task execution
         """
-        self.log_task_result('async failed', result)
-
-    @log_exceptions
-    def v2_playbook_on_no_hosts_matched(self):
-        """
-        Implementation of the callback endpoint to be
-        fired when a playbook finds no hosts to execute on.
-        """
-        self.write_log('No hosts matched.')
-
-    @log_exceptions
-    def v2_playbook_on_no_hosts_remaining(self):
-        """
-        Implementation of the callback endpoint to be
-        fired when a playbook has no hosts remaining
-        to execute on.
-        """
-        self.write_log('No hosts remaining.')
+        logger.error()
 
         # TODO: unimplemented methods follow - do we need them?
```

Comment:
Today let's just put a `logger.info()` in these and make sure we implement every upstream callback.